code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -188,6 +188,8 @@
 		else*/
 		msg += "[T.He] [T.is] wearing \icon[wear_id] \a [wear_id].\n"
 
+	if(attempt_vr(src,"examine_weight",args)) return //VOREStation Code

```

Comment:
This should be, instead,
msg += attempt_vr(src,"examine_weight",args) //VOREStation Code

So the value returned (the string of text) from examine_weight is added to the message here.